packages/engine/Source/Shaders/DepthPlaneVS.glsl

```diff
@@ -5,6 +5,8 @@ out vec4 positionEC;
 void main()
 {
     positionEC = czm_modelView * position;
+    vec4 positionEC = czm_modelView * position;
+    gl_Position = czm_projection * positionEC;
```

Comment:
I don't think we need this change. Now we are redeclaring `positionEC`, and defining `gl_Position` twice.